challenge3/webapp/yukaina/app/services/monthly_energy_logs_import_service.rb

```diff
@@ -0,0 +1,16 @@
+class MonthlyEnergyLogsImportService
+  def initialize(csv_row:)
+    @csv_row = csv_row
+  end
+
+  def call
+    ActiveRecord::Base.transaction do
+      monthly_label = MonthlyLabel.find_or_create_by!(id: @csv_row[:monthly_label_id], year: @csv_row[:year], month: @csv_row[:month])
+      house = House.find(@csv_row[:house_id])
+      MonthlyEnergyLog.create!(
+        @csv_row.to_h.slice(:id, :temperature, :daylight, :production_volume).
+          merge(monthly_label: monthly_label, house: house, family: house.family),
```

Comment:
```suggestion
          merge(monthly_label: monthly_label, house: house, family: house.family)
```
好みの問題かもしれていが、最後の `,` は不要

Reply:
そうですね。自分もナシ最初書いていなかったのですが、onkcop で指摘されて...

> # 見た目がアレだが、ES2017 でも関数引数のケツカンマが許容されるので
> # 世界はそちらに向かっている。
とのことでした。
https://github.com/onk/onkcop/blob/979b8e74eda394434e49caaee31d3f3d781d14f6/config/rubocop.yml#L383-L391

とは言っても、チームとかプロジェクトで揃っていれば良いですよね。

Reply:
マジか！！！！

なんとなく、気持ち悪いと思うオレは老害なのだろうか。
今後、気にしないようにします。